h/auth/services.py

```diff
@@ -7,13 +7,17 @@
 from pyramid_authsanity import interfaces
 import sqlalchemy as sa
 from zope import interface
-from zope.sqlalchemy import mark_changed
 
 from h.models import AuthTicket
 from h.auth.util import principals_for_user
 
 TICKET_TTL = datetime.timedelta(days=7)
 
+# We only want to update the `expires` column when the tickets `expires` is at
+# least one minute smaller than the potential new value. This prevents that we
+# update the `expires` column on every single request.
+NEW_EXPIRY_THRESHOLD = datetime.timedelta(minutes=1)
```

Comment:
I know it's pernickety, but `NEW_EXPIRY_THRESHOLD` doesn't make a lot of sense to me as a name. Perhaps `TICKET_REFRESH_INTERVAL`?